server/src/main/resources/application.properties

```diff
@@ -15,7 +15,7 @@ spring.application.name=conductor
 springdoc.api-docs.path=/api-docs
 loadSample=true
 
-conductor.db.type=redis_standalone
+conductor.db.type=memory
 conductor.queue.type=redis_standalone
```

Comment:
This was a test to force some other classes to load - it should really be overwritten by users though.